_posts/2018-03-30-Introducing-Bazel-Cquery.md

```diff
@@ -17,22 +17,20 @@ The query commands also support a set of flag-based options that govern query be
+What’s the difference? `query` doesn’t understand build flags and returns all possible answers to a given query expression. `cquery` (configurable query) runs at a later point in the build process, after flag evaluation and [configurable attribute](https://docs.bazel.build/versions/master/be/common-definitions.html#configurable-attributes) resolution. Thus, it can understand [build options](https://docs.bazel.build/versions/master/command-line-reference.html#command-line-reference) and give the answer to a specific Bazel invocation as dictated by its set of flags. Since it runs at a later point, `cquery` is by nature slower than `query`.  
 
 `cquery` has yet to reach feature parity with `query`, but as we develop it further, `cquery` will also be able to expose information to which `query` does not have access. See the comparison below for what’s currently supported. 
 
-|                      | `query`                                                                           |`cquery`                                | both                                                                            |
-|----------------------|-----------------------------------------------------------------------------------|----------------------------------------|---------------------------------------------------------------------------------|
-| Performance          | faster, less acurate                                                              | slower, accurate                       |                                                                                 |
-| Functions            | siblings, buildfiles, tests                                                       | config                                 | allpaths, attrs, dep, filter, kind, labels, loadfiles, rdeps, somepath, visible |
-| Output Formats       | build, label,  label_kind, minrank, maxrank, location, package, graph, xml, proto | label_and_configuration, transitions** |                                                                                 |
-| Options              | query options                                                                     | cquery options, build options          | common query options                                                            |
+|                      | `query`                                                                    |`cquery`                              | both                                                                            |
```

Comment:
There is no need to line up the table in the Markdown sources, which makes it easier to maintain in the future. E.g. the following is OK:

```
| Foo | Bar | Baz | Long Column |
| --- | --- | --- | --- |
| This is a very long cell | . | or not | |
```

Note how the text width nor the dashes need to match (but I think you need at least 3 dashes per cell after the header row).